modules/mobile/app/controllers/mobile/v0/letters_controller.rb

```diff
@@ -8,13 +8,30 @@
 module Mobile
   module V0
     class LettersController < ApplicationController
+      DOWNLOAD_PARAMS = %i[
+        militaryService
+        serviceConnectedDisabilities
+        serviceConnectedEvaluation
+        nonServiceConnectedPension
+        monthlyAward
+        unemployable
+        specialMonthlyCompensation
+        adaptedHousing
+        chapter35Eligibility
+        deathResultOfDisability
+        survivorsAward
+      ].freeze
```

Comment:
I was going to suggest using the list of letters available in the mobile letters model but it looks like it's not the same list. is this just a fact of the upstream service or is there something else going on?

Reply:
Yeah, these are different than the letter types. These are options. EVSS uses the same ones. I believe they impact the data included in the letter. I'm going to test that. These are listed in the API docs and I copied this list from the letter generator controller.